components/brave_wallet/renderer/resource_helper.cc

```diff
@@ -5,7 +5,11 @@
 
 #include "brave/components/brave_wallet/renderer/resource_helper.h"
 
+#include "base/base64.h"
+#include "base/memory/ref_counted_memory.h"
+#include "third_party/skia/include/core/SkBitmap.h"
```

Comment:
SkBitmap is included in the ui/gfx/image/image.cc, so we have "calling 'AsBitmap' with incomplete return type 'SkBitmap'" without this include

Reply:
but we are not using the return value and we just pass the return value into `EncodeBGRASkBitmap`. Unless we access any members in SkBitmap, otherwise I don't think we need that header.

Reply:
I've just tried to switch #include "third_party/skia/include/core/SkBitmap.h" -> "ui/gfx/image/image.h"
I think we need complete type here since it returns value, not a pointer.

Reply:
ok, that makes sense for value return.